src_assets/common/assets/web/public/assets/locale/en.json

```diff
@@ -243,9 +243,9 @@
     "origin_web_ui_allowed_lan": "Only those in LAN may access Web UI",
     "origin_web_ui_allowed_pc": "Only localhost may access Web UI",
     "origin_web_ui_allowed_wan": "Anyone may access Web UI",
-    "output_name_desc_unix": "During Sunshine startup, you should see the list of detected displays. Note: You need to use the id value inside the parenthesis.",
+    "output_name_desc_linux": "During Sunshine startup, you should see the list of detected displays. Note: You need to use the id value inside the parenthesis.",
     "output_name_desc_win": "Manually specify a display to use for capture. If unset, the primary display is captured. Note: If you specified a GPU above, this display must be connected to that GPU. The appropriate values can be found using the following command:",
-    "output_name_unix": "Display number",
+    "output_name_linux": "Display number",
```

Comment:
This should apply to both Linux and macOS now, since https://github.com/LizardByte/Sunshine/pull/2449

Can the platform-i18n.js handle this case? Assuming there will be more unix versus windows cases in the future.

Reply:
Currently, without changing anything I'm using `$tp('output_name', $t('output_name_linux'))` to do this, as it fails to find the key for macOS, I gave it the linux value as default, but we probably could add a rule for that directly into the platform-i18n.

 1. if we can't find a key for `macos` we take `linux`
 2. if we can't find a key for either `macos` or `linux` we take a key for `unix`
 
 Which option do you prefer?

Reply:
I think option 2 if I understand it correctly.

First check for the OS is in the key, and if not found look for the `unix` version.

Reply:
@ReenigneArcher Reverted those key names, and fixed the platform-i18n, I think it's better now, lemme know